<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageGradesPage</class>
 <widget class="QWidget" name="ManageGradesPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>225</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>451</width>
     <height>210</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_headline">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>451</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Manage Grades</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_return">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>431</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Return</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_studentId">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Student ID</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_confirm">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>90</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Add/Change</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_subjectName">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>201</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Subject Name</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_grade">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>90</y>
      <width>82</width>
      <height>28</height>
     </rect>
    </property>
    <property name="maxVisibleItems">
     <number>10</number>
    </property>
    <property name="maxCount">
     <number>9</number>
    </property>
    <property name="insertPolicy">
     <enum>QComboBox::InsertPolicy::NoInsert</enum>
    </property>
    <property name="minimumContentsLength">
     <number>2</number>
    </property>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2.5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3.5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4.5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_subjectType">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
    <property name="maxVisibleItems">
     <number>10</number>
    </property>
    <property name="maxCount">
     <number>9</number>
    </property>
    <property name="insertPolicy">
     <enum>QComboBox::InsertPolicy::NoInsert</enum>
    </property>
    <property name="minimumContentsLength">
     <number>2</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_removeGrade">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>120</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
